string recommended_drink
bool customer_opinion